src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -298,7 +298,9 @@ void
 FixedwingAttitudeControl::vehicle_manual_poll()
 {
 
-	if (_vcontrol_mode.flag_control_manual_enabled && !_vehicle_status.is_rotary_wing) {
+	// If VTOL, generate rates sp with FW attutide controller if in transition or in fixed-wing flight
```

Comment:
`attutide` -> `attitude`.

Also, this is not really readable or understandable.